climada/hazard/tc_tracks.py

```diff
@@ -1509,6 +1566,10 @@ def ibtracs_fit_param(explained, explanatory, year_range=(1980, 2019), order=1):
     match = (years >= year_range[0]) & (years <= year_range[1])
     ibtracs_ds = ibtracs_ds.sel(storm=match)
 
+    if "wind" in variables:
+        for agency in IBTRACS_AGENCIES:
+            ibtracs_ds[f'{agency}_wind'] *= IBTRACS_AGENCY_1MIN_WIND_FACTOR[agency]
+
```

Comment:
This makes sure that wind speeds are scaled correctly during the fitting of parameters (for the `_estimate_vmax` and `_estimate_pressure` functions). This changes the fit parameters and a lot of unit tests need to be adjusted, but the numerical changes are small.

Reply:
Well done, this could have easily been overlooked...

Reply:
But now if you call `read_ibtracs_netcdf` with `rescale_windspeed=False` and the provider you choose has 10-min wind, will you not then gap-fill wind speed based on a fit that represents 1-min and not 10-min wind?

Reply:
Well spotted, I added a warning and mentioned it in the docstring:
https://github.com/CLIMADA-project/climada_python/blob/6cb1ee7114d64d8f330a7f9e216506530b9b41ed/climada/hazard/tc_tracks.py#L359-L372